### PR TITLE
chore: v0.5.0 release

### PR DESCRIPTION
**Pull request by ivan-savciuc**

### CODE REVIEW

**Review by Serpentiel — DISMISSED** [review]
lgtm

File: charts/aiven-operator/values.yaml [hunk 1]
```diff
@@ -6,6 +6,10 @@ replicaCount: 1
 
 nameOverride: ""
 fullnameOverride: ""
+defaultToken: ""
+metricsBindAddress: ""
```
Comment (mhoffm-aiven) [inline charts/aiven-operator/values.yaml:10]: defaults seem broken here; those values will probably lead to a broken command being executed right?

File: charts/aiven-operator/Chart.yaml [hunk 2]
```diff
@@ -3,7 +3,7 @@ name: aiven-operator
 description: A Helm chart to deploy the aiven operator
 type: application
 version: v0.4.1
```
Comment (mhoffm-aiven) [inline charts/aiven-operator/Chart.yaml:5]: also 0.5.0 here? configuration changed slightly

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,14 @@
 # Changelog
 
 ## [MAJOR.MINOR.PATCH] - YYYY-MM-DD
-+ add `webhooks.containerPort` value
+
+## aiven-operator-v0.5.0 - 2022-11-28
+
+- Add ability to link resources through the references
```
Comment (mhoffm-aiven) [inline CHANGELOG.md:7]: i would just go with "sync with aiven-operator v0.5.0" ( and add the new configuration options here )

File: charts/aiven-operator/ci/test-values.yaml [hunk 4]
```diff
@@ -1,2 +1,3 @@
 webhooks:
   enabled: false
+defaultToken: test123
```
Comment (mhoffm-aiven) [inline charts/aiven-operator/ci/test-values.yaml:3]: better dont do this

File: charts/aiven-operator/values.yaml (hunk 1 above)
Comment (ivan-savciuc) [reply charts/aiven-operator/values.yaml:9]: nope, in case it is empty these feature gets disabled

**Review by mhoffm-aiven — APPROVED** [review]
lgtm

**Pull request merged**